src/Map/Map.stories.tsx

```diff
@@ -199,3 +200,14 @@ HIIPointColour.args = {
   pointStrokeColor: '#8a8988',
   pointStrokeWidth: 1,
 }
+
+export const HIIPointZoom = Template.bind({})
+HIIPointZoom.args = {
+  sourceJson: hiiJson,
+  pointExpression: pointColourExpression,
+  pointRadiusExpression: ['interpolate', ['linear'], ['zoom'], 1, 5, 10, 10],
+  pointStrokeColor: '#8a8988',
+  pointStrokeWidth: 1,
+  zoomLocation: [-3.5, 55],
```

Comment:
would it be possible to like create a button that updates prop just for a storybook, maybe even generate 
https://docs.mapbox.com/mapbox-gl-js/example/flyto/ just wonder.